doc/source/install.rst

```diff
@@ -13,7 +27,14 @@ To install the latest development version, do
 
     pip3 install --user git+https://github.com/greenplum-db/GreenplumPython
 
+NOTE: This version is considered UNSTABLE. DON'T use it in the production environment! Stable version will be released soon.
+
+GreenplumPython requires [plpython3](https://docs.vmware.com/en/VMware-Tanzu-Greenplum/6/greenplum-database/GUID-analytics-pl_python.html) 
+extension to be installed on Greenplum/Postgres.
 
-NOTE: This version is considered UNSTABLE. DON'T use it in the production environment!
+[dill](https://github.com/uqfoundation/dill) as an optional dependency for GreenplumPython `plpython` side, 
+provides convenient features like auto-importing modules in the `plpython` functions. 
+For better use of greenplumPython we'd better install `dill` on the server side, and if you want to use third party packages you'll also need to install the package on the server side.
```

Comment:
rephrase to:

To install `dill` or any other python modules on the `plpython` side, refer to [GPDB plpython document](https://docs.vmware.com/en/VMware-Tanzu-Greenplum/6/greenplum-database/GUID-analytics-pl_python.html#pip39) for more details.


Avoid sentences like "we'd better xxx"/"you should xxx" in this document.